### PR TITLE
Placeholderify README

### PR DESCRIPTION
**Pull request by ZeoNight**
Start the process of making this a actual fork.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,32 +11,32 @@ If you want to read about why this fork was created, check out [our FAQ page](ht
 
 # Installation
 
-- All downloads and instructions for PolyMC can be found [here](https://polymc.org/download/)
-- Last build status: <https://github.com/PolyMC/PolyMC/actions>
+- All downloads and instructions for PolyMC can be found [here](https://todo)
```
Comment (Weredime) [inline README.md]: Remove this probably

File: README.md [hunk 2]
```diff
@@ -11,32 +11,32 @@ If you want to read about why this fork was created, check out [our FAQ page](ht
...
 
-There are per-commit development builds available [here](https://github.com/PolyMC/PolyMC/actions). These have debug information in the binaries, so their file sizes are relatively larger.
+There are per-commit development builds available [here](https://github.com/PlaceholderMC/PlaceHolderMC/actions). These have debug information in the binaries, so their file sizes are relatively larger.
 Portable builds are provided for AppImage on Linux, Windows, and macOS.
 
 For Debian and Arch, you can use these packages for the latest development versions:  
-[![polymc-git](https://img.shields.io/badge/aur-polymc--git-blue)](https://aur.archlinux.org/packages/polymc-git/)
-[![polymc-git](https://img.shields.io/badge/mpr-polymc--git-orange)](https://mpr.makedeb.org/packages/polymc-git)  
+[![polymc-git](https://img.shields.io/badge/aur-polymc--git-blue)](https://aur.archlinux.org/packages/todo)
+[![polymc-git](https://img.shields.io/badge/mpr-polymc--git-orange)](https://mpr.makedeb.org/packages/todo)
```
Comment (Weredime) [inline README.md]: remove this probably

File: README.md [hunk 3]
```diff
@@ -47,23 +47,23 @@ If there are any issues with the space or you are using a client that does not s
 
 We also have a subreddit you can post your issues and suggestions on:
 
-[r/PolyMCLauncher](https://www.reddit.com/r/PolyMCLauncher/)
+[r/PlaceholderMCLauncher](https://www.reddit.com/r/PlaceholderMCLauncher/)
```
Comment (Weredime) [inline README.md]: Non-existent. https://www.reddit.com/r/PlaceholderMC/ is taken by me, if anybody wants it

**Review by Weredime — APPROVED** [review]
sure, but im not a maintainer here, just helping

**Comment by BowDown097** [comment]
sneed

**Comment by Weredime** [comment]
> sneed

no u

**Comment by Iotados** [comment]
Hey, sorry to intrude but because some people may not entirely have the know-how of what "unistall" means, perhaps just a pinned issue on it? 

Just so people asking can be directed to the issue in question and be given explicit instructions by people who know what they're talking about. I know it may sound like a dumb thing but hey, it could be useful to some.

File: README.md [hunk 4]
```diff
@@ -47,23 +47,23 @@ If there are any issues with the space or you are using a client that does not s
...
 
 ## Building
 
-If you want to build PolyMC yourself, check [Build Instructions](https://polymc.org/wiki/development/build-instructions/) for build instructions.
+If you want to build PolyMC yourself, check [Build Instructions](https://todo/wiki/development/build-instructions/) for build instructions.
 
 ## Translations
 
-The translation effort for PolyMC is hosted on [Weblate](https://hosted.weblate.org/projects/polymc/polymc/) and information about translating PolyMC is available at <https://github.com/PolyMC/Translations>
+The translation effort for PolyMC is hosted on [Weblate](https://hosted.weblate.org/projects/placeholdermc/placeholdermc/) and information about translating PolyMC is available at <https://github.com/PolyMC/Translations>
```
Comment (jonatjano) [inline README.md:62]: You missed one here
```suggestion
The translation effort for PolyMC is hosted on [Weblate](https://hosted.weblate.org/projects/placeholdermc/placeholdermc/) and information about translating PlaceholderMC is available at <https://github.com/PolyMC/Translations>
```

**Comment by Scrumplex** [comment]
I think this isn't needed anymore. Thanks anyway!

**Pull request closed without merging**